[CUBRIDMAN-262] Add HA trouble-shooting manual for HA failover / failback message (#578)
CUBRID/cubrid-manual: en/admin/troubleshoot.rst

  
If fail-over or fail-back occurs, a node changes its role.
  

> The following is the log file of the cub_master that is changed as slave node after fail-back or master node after fail-over; log messages for fail-over and fail-back are prefixed with [Failover] or [Failback].

Reviewer: ```suggestion
The following is the log file of the cub_master that changes to a slave node after fail-back or to a master node after fail-over; log messages for fail-over and fail-back are prefixed with [Failover] or [Failback].
```

Author: 수정하였습니다.

    Node event: [Failover] [Diagnosis] The master node (testhost01) has lost its role due to server process problem, such as disk failure.

    Time: 01/09/25 16:14:13.614 - ERROR *** file /../../src/executables/master_heartbeat.c, line 1179 ERROR CODE = -988, Tran = -1, EID = 7
    Node event: [Failover] [Success] Current node has been successfully promoted to master.

    Time: 01/09/25 16:14:13.614 - ERROR *** file ../../src/executables/master_heartbeat.c, line 1196 ERROR CODE = -988, Tran = -1, EID = 8
    Node event: HA Node Info
    ================================================================================
    * group_id : hagrp   host_name : testhost02   state : master
    --------------------------------------------------------------------------------
    name                 priority   state           score      missed heartbeat
    --------------------------------------------------------------------------------
    testhost02           2          to-be-master    -4094      0
    testhost01           1          unknown         32767      1
    ================================================================================

> Above example is an information which is printed out to the cub_master log; it is the process that the 'testhost02' host changes the role from slave to master by fail-over because the master node 'testhost01' has lost its role due to server process problem.

Reviewer: ```suggestion
The above example shows information printed in the cub_master log; it describes the process where the 'testhost02' host changes its role from slave to master during a fail-over because the master node 'testhost01' lost its role due to a server process issue.
```

Author: 수정하였습니다.
